app/assets/stylesheets/components/vacancies-count.scss

```diff
@@ -0,0 +1,14 @@
+.vacancies-count {
```

Comment:
I notice there is something off with the alignment here. Looks like a combination of tabs and spaces, with double the needed spaces.

We follow the Ruby styleguide which prefers spaces to tabs https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide

I'd recommend setting your editor to automatically set these the correct indentations for you.

![Uploading Screenshot 2019-03-12 at 14.58.43.png…]()

Reply:
Thanks for the tips. I believe I've fixed it with my latest commit now. Please let me know if I've made the same mistakes on other files.

Reply:
Sorry for messing around again. It looked fixed in my Sublime for the previous commit but when I look at the file here again it's not! I just did the indentation again for the entire file and this time it looks fine now. Sorry about that!